systemtests/volsupervisor_test.go

```diff
@@ -140,27 +115,24 @@ func (s *systemtestSuite) TestVolsupervisorRestart(c *C) {
 
 	volName := genRandomVolume()
 	fqVolName := fqVolume("policy1", volName)
-
 	c.Assert(s.createVolume("mon0", fqVolName, map[string]string{"unlocked": "true"}), IsNil)
 
-	time.Sleep(4 * time.Second)
```

Comment:
Several changes were made to the systemtests.

Two tests weren't actually testing anything, that was one of them. 
(volumes now don't snapshot at all when the volume is mounted, but the 
test never mounted a volume so the snapshots were "working" but not 
correctly) Modifications were made to the policy to accommodate them, 
which cascaded into other changes.

On 22 Aug 2016, at 14:13, Bill Robinson wrote:

>>  	c.Assert(s.createVolume("mon0", fqVolName, 
>> map[string]string{"unlocked": "true"}), IsNil)
>>
>> -	time.Sleep(4 * time.Second)
>
> What was the reason for this big change in sleep time?  I'm having 
> trouble seeing it from the diff
>
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/volplugin/pull/435/files/4b967abbd644e9f998ece77e43e30813f6ccde77..f2e5743d552dd2415c24d3efcdd4f70cc7759f41#r75759991